QUARKUS-278.md

```diff
@@ -0,0 +1,54 @@
+# QUARKUS-278 - Test and verify Integration with RH-SSO 7.4
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-278
+
+Coverage should ensure the integration with RH-SSO 7.4 and Quarkus security features.
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+Following actions were taken to ensure test coverage and automation for native image:
+ - Define the scope of the testing
+ - Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+ - Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+ - Determine impact on QE infrastructure - e.g. increased complexity of existing automation, maintenance burden, need for new external resources
+ - Check a need for involvement of other product/component teams
+
+### Impact on testsuites and testing automation:
```

Comment:
Also, I've added some scenarios to cover more integrations in the beefy project like for example with OIDC and Rest Client: https://github.com/quarkus-qe/beefy-scenarios/pull/54
But I'm not sure if we should or if it's worthy to be noted here.

Reply:
Added